src/tests/utils/convert.js

```diff
@@ -202,14 +202,12 @@ describe('convert render functions', () => {
 
       return convert.getHeader(wbDef)
         .then(string => {
-          string.should.eql(`const maybeIncludeAuth = (request, z, bundle) => {
+          string.trim().should.eql(`const maybeIncludeAuth = (request, z, bundle) => {
```

Comment:
`.trim()` is to remove leading and trailing spaces generated by the template. I leave the code unformatted because, instead of formatting the code in the templates, I think I'll make another PR that uses [Prettier](https://prettier.io) to format the generated code.